widgets/glade/AnacondaWidgets.xml

```diff
@@ -142,14 +142,34 @@
             </properties>
         </glade-widget-class>
 
-    <glade-widget-class title="Layout Indicator"
-			            name="AnacondaLayoutIndicator"
-			            icon-name="widget-gtk-label"
-			            generic-name="AnacondaLayoutIndicator">
-      <properties>
-        <property id="label-width" default="12" />
-      </properties>
-	</glade-widget-class>
+        <glade-widget-class title="Layout Indicator"
+                            name="AnacondaLayoutIndicator"
+                            icon-name="widget-gtk-label"
+                            generic-name="AnacondaLayoutIndicator">
+            <properties>
+                <property id="label-width" default="12" />
+            </properties>
+        </glade-widget-class>
+
+        <glade-widget-class title="Spinner"
+                            name="AnacondaSpinner"
+                            icon-name="widget-gtk-spinner"
+                            generic-name="AnacondaSpinner"
+                            since="3.2">
+            <properties>
+                <!-- Remove properties inherited from GtkImage -->
+                <!-- Setting disabled doesn't hide them in the glade interface
+                     (nor does setting visible, oddly), but it keeps them being
```

Comment:
Yes. The spinner derives from GtkImage, but none of the GtkImage properties should be applied to the spinner.